todo_app/app.py

```diff
@@ -32,30 +32,17 @@ def create_app():
     client_id = os.getenv('Git_Client_ID')
     client = WebApplicationClient(client_id)
```

Comment:
The code here suggests that your app should be checking authentication, but I can access your app in Azure without it; do you know why that is? We should ensure the latest build of your code is actually running in Azure

Reply:
Added another OAuth for Azure